jupyter/datascience/ubi9-python-3.12/pyproject.toml

```diff
@@ -27,6 +27,7 @@ dependencies = [
 
     # JupyterLab packages
     "odh-elyra==4.2.3",
+    "odh-jupyter-trash-cleanup==0.1.0",
```

Comment:
Run

    make refresh-pipfilelock-files

that will update the pylock.toml files next to the pyproject.toml files.

Commit the updated pylock.toml files.